### PR TITLE
deheader: fix multiple spelling errors

### PR DESCRIPTION
**Pull request by ndowens**
While here, correct order of variables and fix license format

### CODE REVIEW

File: srcpkgs/deheader/patches/0003-Typo-fix.patch [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ heads off the most common problems by suppressing deletion of headers
+ that are required for portability, but your mileage may vary.</para>
+ 
+-<para>The depenedency scanner does not ignore the text of comments.
++<para>The dependency scanner does not ignore the text of comments.
+ This, e.g, a reference to "log10" in a comment will produce a spurious
+ warning that &lt;math.h&gt;  is required for portability.</para>
+ 
+-- 
+2.20.1
```
Comment (Vaelatern) [inline srcpkgs/deheader/patches/0003-Typo-fix.patch]: Can't this patch come from upstream at the next release? Not exactly critical bugfix for functionality.

File: srcpkgs/deheader/patches/0001-Fix-spelling-error-in-manpage.patch [hunk 2]
```diff
@@ -0,0 +1,25 @@
+From e455d0a71612cdf614afaceaef10822454434d33 Mon Sep 17 00:00:00 2001
+From: Reiner Herrmann <reiner@reiner-h.de>
+Date: Sat, 23 Sep 2017 13:55:45 +0200
+Subject: [PATCH 1/3] Fix spelling error in manpage
```
Comment (Vaelatern) [inline srcpkgs/deheader/patches/0001-Fix-spelling-error-in-manpage.patch]: Not exactly mission critical bugfix :)

Comment (ndowens) [reply srcpkgs/deheader/patches/0001-Fix-spelling-error-in-manpage.patch]: :) yep, but still nice to fix some "issues"

**Comment by Vaelatern** [comment]
Are these all the patches since the last release?

**Comment by ndowens** [comment]
Yep a few commits after last release

**Comment by ndowens** [comment]
Closing, as it is not critial

**Pull request closed without merging**